lib/westfield_datetime_helper.rb

```diff
@@ -1,2 +1,15 @@
 require "westfield_datetime_helper/version"
 require "westfield_datetime_helper/matchers/zulu_time"
+require "westfield_datetime_helper/validators/zulu_time"
```

Comment:
Why is this prefixed by westfield? Given that it is open source i think we should remove all references to westfield. Does not really add anything. Also the module is WestfieldLabs so if it is doing to be kept then the files should be renamed to include `labs`

If datetime_helper is a bit generic perhaps a `zulu` prefix would be better